js/schema/style.json

```diff
@@ -27,9 +27,23 @@
       "format": "color",
       "maxLength": 256
     },
+    "border-color-hover": {
+      "type": "string",
+      "format": "color",
+      "maxLength": 256
+    },
     "border-radius": {
       "type": "number"
     },
+    "font-family": {
+      "type": "string",
+      "format": "text",
+      "maxLength": 256
+    },
+    "font-size": {
```

Comment:
@ohadBrunner  hey! I think we should be using the existing `size` property to set this to fixed sizes instead of new prop `font-size` because font size number will mean different things on different platforms. Similar to how we are doing to the carousel buttons.

Reply:
fixed.